front/src/applications/stdcm/views/StdcmRequestModal.jsx

```diff
@@ -33,25 +29,35 @@ import { Spinner } from 'common/Loader';
 import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 
 export default function StdcmRequestModal(props) {
-  const { t } = useTranslation(['translation', 'operationalStudies/manageTrainSchedule']);
+  const { t } = useTranslation([
+    'translation',
+    'operationalStudies/manageTrainSchedule',
+    'translation',
+  ]);
   const osrdconf = useSelector(getConf);
   const dispatch = useDispatch();
-  const [postStdcm] = osrdEditoastApi.usePostStdcmMutation();
+
+  const [postStdcm] = osrdEditoastApi.endpoints.postStdcm.useMutation();
```

Comment:
why do you change this ?

Reply:
I find this clearer as you can directly access the endpoint in osrdEditoastApi by clicking on postStdcm, what do you think ?

Reply:
it even seems to provide better types infos. yeah keep this one

![Screenshot_20230822_154653](https://github.com/osrd-project/osrd/assets/24652545/301534f5-bc9c-401f-bf0c-59602c2702c2)